### PR TITLE
docs(ledger-browser): add initial GUI documentation and fix some bugs

### PR DESCRIPTION
**Pull request by outSH**
- Add new documentation section to project docs. New pages can be found under
    `Cactus` -> `Ledger Browser` category. Documentation include GUI setup
    instructions, application overviews, developer guide and tutorial
    on developing new app plugin for the GUI.
- Do some minor quality of life improvements and bug fixes.
- Add missing GUI database schema file.
- Remove dead code from GUI that still used `getAppList`.
- Add documentation links to the GUI. For now, most links are empty and they
    will be set once this PR is merged and the docs URLs are confirmed.
- Add sample tutorial application (the same that is created step-by-step in
    the tutorial in the documentation)
- Improve plugin app URL regex to allow more valid paths.
- Expose ethereum and fabric schemas in supabase-all-in-one
- Extend persistence plugin init SQL with code for exposing custom schemas.
- Improve supabase-all-in-one readme documentation.
- Fix persistence sample setup scripts ports so that both scripts can be run
    at the same time.

Depends on #3448
Depends on #3449

Signed-off-by: Michal Bajer <michal.bajer@fujitsu.com>

**Pull Request Requirements**
- [ ] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [ ] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [ ] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [ ] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [ ] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

**Review by petermetz — CHANGES_REQUESTED** [review]
@outSH LGTM, I'm just blocking the merge manually until the parent PRs have been merged because right now the bot that does the enforcement of the dependent PRs seems to be dead (I'm working on an alternative solution: https://github.com/hyperledger/governance/issues/314 )

Please re-request review when the parent PRs are merged, no actual change is necessary from my side.

**Review by petermetz — APPROVED** [review]

**Comment by RafaelAPB** [comment]
+1, cool work! This will be handy to the other packages as well

**Review by VRamakrishna — APPROVED** [review]
LGTM. Can we revisit the envisioned integrated architecture (in `ROADMAP.md`) and see where the ledger browser and the supabase fit in? We can open an issue for discussion on this, and I can update the architecture diagram accordingly.

**Pull request merged**
